src/main.ts

```diff
@@ -30,9 +32,81 @@ const getMongoDBClient = async (): Promise<NoSQLWrapper> => {
         const result = await db.collection('users').find({}).toArray();
         return result;
     }
+
+    const FindOneUser = async (email:string, password: string): Promise<any> => {
+        const result = await db.collection('users').findOne({email,password});
+        return result;
+    }
+
+    const CreateArticulo = async (articulo: any): Promise<any> => {
+        const year = await db.collection('years').insertOne({
+            year: articulo.year,
+            disponible: articulo.disponible
+        });
+        const volumen = await db.collection('volumenes').insertOne({
+            volumen: articulo.volumen
+        });
+        // para que no se inserte dos veces en la misma coleccion
+        const yy = {
```

Comment:
the same here you can use yy as a name you need to be clarified with the code, you need to program clean code, and a good selection to named functions and vars is really important